DataManagement/GitTutorial.Rmd

```diff
@@ -196,60 +218,100 @@ git clone git:kingsleyshacklebolt/dragon_study.git
 
-Green: code you have added
 
-Red: code you have deleted
+### Some rules
 
-!!! picture of green and red lines !!!
+Commits are cheap.
+It does not take much time to click on **Commit**, write a few words or sentences about the commit and stage it.
+Therefore, commit often and provide useful messages so you can keep track of what you are doing. 
 
-Commits are cheap. 
-Commit often and provide useful messages so you can keep track of what you are doing. 
-Don’t do this:
+This is a bad example of a workflow:
 <br/><br/>
 ```{r git_commit}
-knitr::include_graphics("Pics/gittutorial/17_git_commit.png")
+knitr::include_graphics("Pics/gittutorial/20_git_commit.png")
 ```
 
-(From xkcd)
+(Picture from xkcd)
 
+### Push and pull
 
-At this stage, all code and changes are still locally stored on your computer. 
-To upload your commits to your remote GitHub repository you need to push (green arrow) these changes to your remote repository on GitHub.
+So far you are still working **locally** on your computer and you have not done anything to the **remote** repository on GitHub.
+All the new code is still **locally** on your computer.
+To upload your commits to your **remote** GitHub repository you need to **Push** (green arrow in the **Git** tab) these changes to your **remote** repository on GitHub.
 
-!!! picture of gir window and green arrow !!!
+<br/><br/>
+```{r git_window2}
+knitr::include_graphics("Pics/gittutorial/21_push.png")
+```
+<br/><br/>
+<br/><br/>
 
 ## Share a repository
 
-!!! picture of 2 people working on same repo !!!
+It is possible to share a repository on GitHub.
+Let's say that Kingsley has a repo on GitHub and wants to collaborate with Angelina on a project.
+Angelina can clone Kingsley's repo locally on her computer, and they can both develop code and push the code to the remote repo on GitHub.
 
-If you are working with other people in a shared repository on GitHub, you will need to pull (blue arrow) to bring their modifications into your local copy of the repository. 
-Do this everytime before you start to work and push your changes regularly to make sure everybody is working on the latest version. 
-However, if you are collaborating on a project we strongly suggest you to use forks and branches (see Collaborating with forks and branches tutorial).
+There are a few things to consider when using this workflow.
+If you are working with other people in a shared repository on GitHub, you will need to pull (blue arrow in the **Git** tab) to bring the modifications your collaborator(s) have made into your local copy of the repository. 
+Do this everytime before you start to work and also push your changes regularly to make sure everybody is working on the latest version.
+This is very important if you are working on the same files.
+
+```{r workflow2}
+knitr::include_graphics("Pics/gittutorial/22_workflow2.png")
```

Comment:
previous fig had solid line for cloning. Should be consistent (I think I like dashed)

Reply:
I was thinking to use dashed line for the cloning which you usually only do once. and then solid lines for the normal workflow. But I also need a figure legend, I guess.

Reply:
don't need a legend as lines are described in figure - just need to be consistent with branch and fork figure

Reply:
oki